helm/providers/charts/cloudant/templates/installPkgCloudantJob.yaml

```diff
@@ -0,0 +1,69 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more contributor
+# license agreements; and to You under the Apache License, Version 2.0.
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: install-package-cloudant
+  namespace: {{ .Release.Namespace | quote }}
+spec:
+  activeDeadlineSeconds: 600
+  template:
+    metadata:
+      name: install-package-cloudant
+    spec:
+      restartPolicy: Never
+      volumes:
+      - name: task-dir
+        configMap:
+          name: install-package-cloudant
+      containers:
+      - name: cloudantpkginstaller
+        image: openwhisk/kube-whisk-script-runner
+        volumeMounts:
+        - name: task-dir
+          mountPath: "/task/myTask.sh"
+          subPath: "myTask.sh"
+        env:
+          - name: "AUTH"
+            valueFrom:
+              secretKeyRef:
+                name: whisk.auth
+                key: system
+          # poorly named envvar; actually expects full apihost (host:port)
+          - name: "APIHOST"
+            valueFrom:
+              configMapKeyRef:
+                name: whisk.config
+                key: whisk_api_host_nameAndPort
+          # for the database used by cloudant package
+          - name: "DB_HOST"
+            valueFrom:
+              configMapKeyRef:
+                name: db.config
+                key: db_host
+          - name: "DB_PORT"
```

Comment:
Add a similar stanza to define DB_PROTOCOL based on db.config/db_protocol

Reply:
Done.